dropwizard-hibernate/src/main/java/io/dropwizard/hibernate/HibernateBundle.java

```diff
@@ -58,7 +58,7 @@ public final void run(T configuration, Environment environment) throws Exception
         environment.healthChecks().register(name(),
                                             new SessionFactoryHealthCheck(
                                                     environment.getHealthCheckExecutorService(),
-                                                    dbConfig.getValidationQueryTimeout().or(Duration.seconds(5)),
+                                                    dbConfig.getValidationQueryTimeout().orElseGet(() -> Duration.seconds(5)),
```

Comment:
This is being called *once*, so I don't think that performance optimization is relevant here.